doc/source/conf.py

```diff
@@ -28,7 +28,7 @@
 
 # necessary when building the sphinx gallery
 pyvista.BUILDING_GALLERY = True
-pyfluent.BUILDING_GALLERY = True
+pyfluent.launcher.fluent_container.DEFAULT_CONTAINER_MOUNT_PATH = pyfluent.EXAMPLES_PATH
```

Comment:
Question: what required this change and the `PYFLUENT_CONTAINER_MOUNT_PATH` specification?  

The tests/functions that require this change will not work when host OS is not the same OS as the container (i.e., both need to be Linux in this case). I don't think this approach is desirable in the long term, ideally we'd make everything work without needing to specify `DEFAULT_CONTAINER_MOUNT_PATH` or `PYFLUENT_CONTAINER_MOUNT_PATH` (except for corner cases).

Reply:
Changed this to use the default mount path

Reply:
I've reinstated this change as passing a container path which is different from examples_path is not working in this case. Note that the Fluent API calls here are almost identical to the parametric unittest in the main repo. I'll revisit this later after the proposed settings API improvement (related to absolute path requirement) on the Fluent side.